windows/NsisInclude/NsisMultiUserLang.nsh

```diff
@@ -41,30 +41,32 @@ Version: 2023-01-30
 	LangString MULTIUSER_LOGON_SERVICE_NOT_RUNNING ${LANG_ENGLISH} "Unable to elevate, Secondary Logon service not running."
 	LangString MULTIUSER_ELEVATION_ERROR ${LANG_ENGLISH} "Unable to elevate, error {ERROR}."
 	; text string for installer section
-	LangString SECTION_CORE_NAME ${LANG_ENGLISH} "Core (Required)"
-	LangString SECTION_CORE_DESCRIPTION ${LANG_ENGLISH} "The main file of the program."
+	LangString SECTION_CORE_NAME ${LANG_ENGLISH} "Core files (required)"
```

Comment:
I think the component description can be modified to be slightly more descriptive.
A) Current component description
B) Suggested component description

A1) Core files - "Install the main file of the program (required)."
B1) This are the basic files needed for ${prodname} to function (required).

A2) Set the options to create shortcuts."
B2) Create shortcuts to ${prodname} on -

A3) "Create the shortcuts in Start menu."
B3) "Create the shortcut in Start menu."

A4) "Install all translation files."
B4) Use ${prodname} in your preferred local system language, install the needed language files.

A5) "Install integrate shredder."
B5) Create an option to delete files from the file context menu in Windows explorer.